### PR TITLE
Change family and column names

### PR DESCRIPTION
**Pull request by steveburton4**

### CODE REVIEW

File: src/main/kotlin/Handler.kt [hunk 1]
```diff
@@ -27,22 +27,24 @@ class Handler : RequestHandler<Request, ByteArray?> {
...
 
     override fun handleRequest(input: Request, context: Context?): ByteArray? {
-        logger.info("Input received for topic ${input.topic} with key ${input.key} and timestamp ${input.timestamp} for delete request ${input.deleteRequest}")
-        val topic = input.topic.toByteArray()
+        logger.info("Input received for topic '${input.topic}' with key '${input.key}' and timestamp '${input.timestamp}' for delete request '${input.deleteRequest}' and table per topic setting of '${input.useTablePerTopic}'")
         val timestamp = input.timestamp
         val isDeleteRequest = input.deleteRequest
         val useTablePerTopic = input.useTablePerTopic
-        val family = HbaseConfig.dataFamily.toByteArray()
+        val family = (if (useTablePerTopic) "cf" else HbaseConfig.dataFamily).toByteArray()
```
Comment (dan-hill2802) [inline src/main/kotlin/Handler.kt:34]: what's `cf`?

Comment (steveburton4) [reply src/main/kotlin/Handler.kt:34]: The column family name used in HTME and HDI with the new paradigm.

Comment (dan-hill2802) [reply src/main/kotlin/Handler.kt:34]: so `cf` = `column family`?

Comment (steveburton4) [reply src/main/kotlin/Handler.kt:34]: Yep

**Review by dan-hill2802 — APPROVED** [review]
looks like all existing functionality is maintained

**Pull request merged**